packages/wellknown-did-verifier/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@sphereon/ssi-sdk-wellknown-did-verifier",
+  "version": "0.6.0",
```

Comment:
develop is at 0.7.x. So a change to 0.7.0 should be enough

Reply:
fixed